src/main/java/com/terraformersmc/terrestria/init/TerrestriaGeneration.java

```diff
@@ -117,6 +119,15 @@ public static void init(BiomeConfig config, Set<String> enabledBiomes) {
 			OverworldBiomes.setRiverBiome(VOLCANIC_ISLAND_BEACH, null);
 			OverworldBiomes.setRiverBiome(VOLCANIC_ISLAND_SHORE, null);
 		}
+
+		if (OUTBACK != null) {
+			OverworldBiomes.addHillsBiome(OUTBACK, OUTBACK, 2);
+			OverworldBiomes.addHillsBiome(OUTBACK, OUTBACK_ULURU, 1);
+			TerrestriaOverworldBiomes.addPredicatedBorderBiome(OUTBACK, OUTBACK_BUSHLAND, b -> {
+				Biome.Category category = b.getCategory();
+				return category != Biome.Category.DESERT && category != Biome.Category.SAVANNA && category != Biome.Category.PLAINS && category != Biome.Category.MESA;
```

Comment:
We should at least check for Calderas here

Reply:
Don't think that's what the point of this check is.